### ISSUE TITLE
Build error: Unknown Escape Sequence

### ISSUE THREAD

**Issue by ffleader1** [revel/examples#56]
On Windows, when I try to run this command in cmd `revel run "%GOPATH%/github.com/revel/examples/chat"`, I got:

    Revel executing: run a Revel application
    WARN  10:07:40    run.go:150: No http.addr specified in the app.conf listening on localhost interface only. This will not allow external access to your application
    C:\Users\Administrator\go\src\github.com\revel\examples\chat\app\tmp\main.go
ERROR 10:07:40 watcher.go:270: Build detected an error                   error="Go Compilation Error (in app\\tmp\\main.go:9): unknown escape sequence"

I have no idea what problem I ran into, until I open the main.go file with an ide and got this:
![image](https://user-images.githubusercontent.com/15016720/61603837-68b8dd00-ac69-11e9-908e-dd2af797ba7c.png)

Imo, to indicate the file location, the syntax should be `github.com/revel/examples/chat/app/tmp/run"`, yet some how revel just change it to `"github.com\revel\examples\chat/app/tmp/run"` all the time, rendering the "Unknown escape sequence" error.

Anyone has any idea how to fix this?

**Comment by Huholoman** [comment]
Im not using windows for a long time, but shouldnt you revel path be like
```
github.com/revel/example/chat/app/tmp/run
``` 
?

**Comment by ffleader1** [comment]
> Im not using windows for a long time, but shouldnt you revel path be like
> 
> ```
> github.com/revel/example/chat/app/tmp/run
> ```
> 
> ?

Yes it should, but that main.go is something generated by revel, as i said at the beginning of the file `GENERATED CODE- DO NOT EDIT`, so if it's anything, it's really revel problem.
